resources/views/livewire/index.blade.php

```diff
@@ -0,0 +1,39 @@
+@php use SweetAlert2\Laravel\Swal; @endphp
+<script type="module">
+    let Swal;
+    const getSweetAlert2 = async () => {
+        // If SweetAlert2 is already loaded, use it
+        if (window.Swal) {
+            return window.Swal;
+        }
+
+        // Otherwise, dynamically import it from CDN
+        try {
+            return (await import('https://cdn.jsdelivr.net/npm/sweetalert2@11/dist/sweetalert2.esm.all.min.js')).default;
+        }
+
+            // Fallback in case of an error (e.g., network issues)
+        catch (error) {
+            console.error('Failed to load SweetAlert2:', error);
+            return {
+                fire: () => {
+                }
+            };
+        }
+    };
+    @if(session()->has(Swal::SESSION_KEY))
+    (async () => {
+        Swal = await getSweetAlert2();
+        Swal.fire(@json(session(Swal::SESSION_KEY)));
+    })();
+    @endif
+
+    window.addEventListener({{ Swal::SESSION_KEY }}, async (event) => {
```

Comment:
I didn't test this manually yet, but I think this will be rendered as

```js
window.addEventListener(sweetalert2, async (event) => {
```

without quotes, and what we want is

```js
window.addEventListener('sweetalert2', async (event) => {
```

```suggestion
    window.addEventListener('{{ Swal::SESSION_KEY }}', async (event) => {
```